### ISSUE TITLE
what's the size of test set

### ISSUE THREAD

**Issue by kgarg8** [ygorg/KPTimes#5]
Table 1 in the paper shows test set as `10k` but when we downloaded and checked, it amounted to `20k`. Could you please double check the figures?

Thanks.

**Comment by ygorg** [comment]
Hi,
Both numbers are correct. The size of the testset of KPTimes is 20k documents. Inside those 20k documents, 10k are from the NYTimes and 10k are from the JapanTimes.

The goal of including those 2 sources is to evaluate the generalizability of the models using documents similar of the training data, but not from the same distribution.
In your experiments you should report the evaluation using the 20k documents, and if needed for a finer grain show the NYTimes and the JPTimes score.
